tests/unit/Debugger/MatchingFileIteratorTest.php

```diff
@@ -0,0 +1,53 @@
+<?php
+/**
+ * Copyright 2018 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace Google\Cloud\Tests\Unit\Debugger;
+
+use Google\Cloud\Debugger\MatchingFileIterator;
+use PHPUnit\Framework\TestCase;
+
+/**
+ * @group debugger
+ */
+class MatchingFileIteratorTest extends TestCase
+{
+    public function testFindsMany()
+    {
+        // There are many files that end in Connection/RestTest.php in the test/unit folder
```

Comment:
It's probably OK, but strictly saying, this test can be broken when someone change those file names.

Reply:
True, I was trying to test that it will indeed find all the files that match the pattern without creating a bunch of extra test files.